core/server/models/base/index.js

```diff
@@ -543,7 +543,7 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
 
         return result;
     }
-
+    
```

Comment:
This change shouldn't be here 😉  - also travis didn't pick this PR up again for some reason, so if you change this and push, hopefully it will happen this time!